compiler/rustc_middle/src/query/mod.rs

```diff
@@ -719,20 +719,20 @@ rustc_queries! {
     ///                         // DefId
     /// trait Trait {           // trait_id
     ///     fn f();             // trait_f
-    ///     fn g() {}           // trait_g
+    ///     fn g() `{}`           // trait_g
```

Comment:
Overzealous replacement. There are more of these below

Reply:
Oops, find and replace was a little to aggressive